### DISCUSSION TITLE
Iso Download

### DISCUSSION THREAD

**Discussion by byt3b4ndit** [Security-Onion-Solutions/securityonion#6947] [General] [answered]
Is there any particular reason this isn't available as a torrent download?
Trying to download the iso with spotty internet is a futile endeavor and it would be much easier as a torrent.

**Comment by dougburks** [comment, marked as answer]
Sorry, we have no plans to offer a torrent. We recommend using non-spotty Internet as that will improve not only your initial download experience but also your update experience.
